LLM/OSS/tools.py

```
@@ -0,0 +1,207 @@
+        return response
+
+
+EMPTY_TOOL_RESULT = ToolResult(name="none", confidence=0.0)
+
+
+def call_submodel(user_text: str) -> str:
+    base = ""
+    try:
+        result = build_answer(user_text, top_k=12)
+        base = (result or {}).get("answer", "").strip()
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)